base/libgit2/diff.jl

```diff
@@ -34,6 +34,26 @@ function diff_tree(repo::GitRepo, oldtree::GitTree, newtree::GitTree)
     return GitDiff(repo, diff_ptr_ptr[])
 end
 
+function GitDiffStats(diff::GitDiff)
```

Comment:
If there isn't a `GitDiffStats` type defined then you can't use it as an annotation in the method definitions below though